config/webpack.config.prod.js

```diff
@@ -215,6 +215,7 @@ module.exports = {
       // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
       'react-native': 'react-native-web',
       'lottie-react-native': 'react-native-web-lottie',
+      'react-native-linear-gradient': 'react-native-web-linear-gradient',
```

Comment:
why is this only in prod?
is this supposed to work out of the box? i dont see it was add to package.json

Reply:
It should be in dev too. I missed it.
These libraries are already added to package.json.
The fix is here - https://github.com/GoodDollar/GoodDAPP/pull/2324